Code/MinimalLib/common.h

```diff
@@ -925,6 +930,9 @@ std::string generate_aligned_coords(ROMol &mol, const ROMol &templateMol,
     match = RDDepict::generateDepictionMatching2DStructure(
         mol, templateMol, confId, refPattern, acceptOrigFailure, false,
         allowRGroups);
+    if (!match.empty()) {
```

Comment:
So this is not wrong, but it's more or less guaranteed to lead to bug reports (also not wrong) for the case where the wedging is within the constrained core: you'll remove the original wedging from those bonds when it was not necessary to do so.

Reply:
True, and I had actually thought about that, then I went for the easy solution. Since now there are at least two people with the same thought, I'll address it properly :)